source/util/AMErrorMonitor.h

```diff
@@ -161,9 +163,11 @@ class AMErrorMon : public QObject {
 	}
 
 	/// Report an alert level AMErrorReport.  Builds an AMErrorReport using the information provided. This function is thread-safe.
-	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "")
+	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "", bool showWarningMessage=false)
```

Comment:
I added this flag for code simplification.` AMErrorMonitor::alert()` will only create an alert message in the error log window. Sometime, we might want a visual alert box popped up. 

I have two usages for that case. That's why I was thinking of making changes here. I can change it back for sure if you are against this.